modules/ztp-install-factory-hubspoke-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="hub-edge-cluster-architecture_{context}"]
+= The hub and edge cluster architecture
```

Comment:
We seem to use edge cluster and spoke cluster interchangeably in the docs.  Can we please teach the user both terms?  In marketing/external facing material, we are using edge cluster, and in the github repo we use spoke clusters.

Reply:
Are we happy to use edge (spoke) cluster. I thought I had named everything edge in docs